examples/resources/cyral_sidecar_listener/resource.tf

```diff
@@ -1,55 +1,55 @@
 resource "cyral_sidecar" "sidecar" {
-  name = "tf-account-sidecar"
+  name              = "tf-account-sidecar"
   deployment_method = "docker"
 }
 
 # Plain listener
 resource "cyral_sidecar_listener" "listener" {
-    sidecar_id = cyral_sidecar.sidecar.id
-    repo_types = ["mongodb"]
-    network_address {
-        host          = "mongodb.cyral.com"
-        port          = 27017
-    }
+  sidecar_id = cyral_sidecar.sidecar.id
+  repo_types = ["mongodb"]
+  network_address {
+    port = 27017
+  }
 }
 
 # Listener with MySQL Settings
 resource "cyral_sidecar_listener" "listener" {
-    sidecar_id = cyral_sidecar.sidecar.id
-    repo_types = ["mysql"]
-    network_address {
-        host          = "mysql.cyral.com"
-        port          = 443
-    }
+  sidecar_id = cyral_sidecar.sidecar.id
+  repo_types = ["mysql"]
+  network_address {
+    # Specify the network address if the sidecar software is deployed on a host with multiple network
+    # interfaces but we want the sidecar to only accept connections on a specific one. Otherwise, if you
+    # leave host empty, the sidecar will listen on all network interfaces.
+    host = "0.0.0.0"
```

Comment:
I think we could omit this and improve the actual docs for the `host` attribute in the `cyral_sidecar_listener` resource. Something along the lines of: `Specify the network interface address that the sidecar needs to listen to in case the sidecar is deployed on a host with multiple network interfaces. If you omit it the sidecar will assume the value "0.0.0.0" and listen on all available network interfaces in the host.`

```suggestion
```